server/tools/hatohol-db-initiator.in

```diff
@@ -132,6 +132,11 @@ def parse_default_conf():
 
 def get_default_conf_item(item_name):
     config = parse_default_conf()
+    if config is None:
+       global not_found_config_file
+       not_found_config_file = True
+       return "<Unknown>"
```

Comment:
Yes, it's an expected behavior.
Although I thought I add code that check if all the necessary options are specified, it would make the source complex.